scalding-serialization-macros/src/main/scala/com/twitter/scalding/serialization/macros/impl/ordered_serialization/TreeOrderedBuf.scala

```diff
@@ -166,19 +178,25 @@ object TreeOrderedBuf {
         case m: MaybeLengthCalculation[_] =>
           val tmpLenRes = freshT("tmpLenRes")
           q"""
-            @inline def noLenCalc = {
-              $noLenCalc
-            }
             @inline def withLenCalc(cnt: Int) = {
```

Comment:
Ahhh.. This was a worry we had before (we could be hampering the jit by making methods too large).

Got it. That's probably a TODO for us: look at logs to see if we are hurting the JIT.